.github/actions/login-to-azure/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Login to Azure
+description: Logs in to Azure using a service principal.
+
+inputs:
+  enable-azure-powershell-session:
```

Comment:
This is an "Az" PS session. It's important, because that's not the same as the "Azure PowerShell module".